### PR TITLE
Deprecate PHP 7.1

### PR DESCRIPTION
**Pull request by MPParsley**
PHP 7.1 is no longer supported, let's remove it.

See https://www.php.net/supported-versions.php

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ dist: trusty
 sudo: false
 
 php:
-  - 7.0
   - 7.1
```
Comment (MPParsley) [inline .travis.yml]:
```suggestion
```

File: composer.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.1",
```
Comment (MPParsley) [inline composer.json]:
```suggestion
        "php": ">=7.2",
```

**Review by RobLoach — APPROVED** [review]
Likely a good call, thanks!

**Comment by webflo** [comment]
Thanks for the PR!

**Comment by PetersRicardo** [comment]
> PHP 7.1 is no longer supported, let's remove it.
> 
> See https://www.php.net/supported-versions.php

This is not true yet, security support is maintained till 01-12-2019 the next drupal release still falls within this time frame (and will support 7.1 as far as I can tell).

Is this intended?

**Comment by goba** [comment]
This is a bit heavy handed. Since this package is the official suggested way to install Drupal with composer (https://www.drupal.org/docs/develop/using-composer/using-composer-to-install-drupal-and-manage-dependencies) it would be useful if the official version constraints would be followed. Drupal 8.8 does not even require PHP 7.1 yet let alone a minimum of 7.2.

**Comment by webflo** [comment]
Thanks for your feedback. I reverted the change and we will use 7.0.8 as the new minimum.

**Pull request merged**